CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+x.x.x Release notes (yyyy-MM-dd)
+=============================================================
+### Enhancements
+* None.
+
+### Fixed
+* If you make a case insensitive query on an indexed string property, it may fail in a way that results in a `No such key` exception if an object that matches the query is deleted. ([realm/realm-cocoa#6830](https://github.com/realm/realm-cocoa/issues/6830), since v6.0.0)
```

Comment:
It's quite wordy, maybe something like:
```suggestion
* A case-insensitive query, on an indexed string property, could throw a `"No such key"` when the query matched a deleted object. ([realm/realm-cocoa#6830](https://github.com/realm/realm-cocoa/issues/6830), since v6.0.0)
```